api/client/commands.go

```diff
@@ -778,14 +778,18 @@ func (cli *DockerCli) CmdStart(args ...string) error {
 			return err
 		}
 	}
-
+	var encounteredError error
```

Comment:
I think it should be `encounteredError `, because it not the last error when start multiple containers.
when start multiple containers, the error message is `Error: failed to start one or more containers` means
encounter a error.  It will never go to line 791 (see below `encounteredError = err`) when start multiple containers because start multiple containers `attach` and `openStdin` should be false.  Only start a container `encounteredError = err` could be execute.